lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,89 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
+        services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+        services.key?('rails') && docker_start(services['rails'], :rails)
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+          services.key?('mysql') && docker_start(services['mysql'], :mysql)
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+          services.key?('postgresql') && docker_start(services['postgresql'], :postgres)
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.docker_start(definitions, service)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for docker_start is too high. [24.37/20] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [17/15] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)

<!-- Comment by [SideCI](https://sideci.com) -->